src/Bindings/Plugin.h

```diff
@@ -74,6 +74,8 @@ class cPlugin
 	virtual bool OnPlayerBrokenBlock        (cPlayer & a_Player, int a_BlockX, int a_BlockY, int a_BlockZ, char a_BlockFace, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta) = 0;
 	virtual bool OnPlayerDestroyed          (cPlayer & a_Player) = 0;
 	virtual bool OnPlayerEating             (cPlayer & a_Player) = 0;
+	virtual bool OnPlayerEditedBook         (cPlayer & a_Player, const cBookContent & a_BookContent, bool a_IsSigned) = 0;
+	virtual bool OnPlayerEditingBook        (cPlayer & a_Player, const cBookContent & a_BookContent, bool a_IsSigned) = 0;
```

Comment:
I assume the book workflow is like this (for example):
User has a book that contains `abc`. They edit it, they type in `def`. As they click "save" or whatever there is in the UI, the client sends the packet to the server requesting the book content change.

Now I, as a plugin author, want to get both the `abc` (the original text) and the `def` (the new text) as a parameter to the "editing" hook; and I want to be able to change the `def` to `ghi` for some reason. The next plugin that gets the hook then receives `abc` as the original text and `ghi` as the new text.